server/build.gradle

```diff
@@ -58,6 +58,22 @@ task generateConfig() {
     }
 }
 
+task apiJar(type: Jar) {
```

Comment:
Please add a comment with context as to why this exists here for others that come back to it in the future.

Reply:
Done!